mirdata/salami.py

```diff
@@ -175,6 +179,17 @@ def sections_annotator_2_lowercase(self):
     def audio(self):
         return librosa.load(self.audio_path, sr=None, mono=True)
 
+    def to_jams(self):
+        return jams_utils.jams_converter(multi_section_data=[(
+                                         [(self.sections_annotator_1_uppercase, 0),
+                                          (self.sections_annotator_1_lowercase, 1)], 'annotator_1'),
+                                         (
+                                         [(self.sections_annotator_2_uppercase, 0),
+                                         (self.sections_annotator_2_lowercase, 1)], 'annotator_2'),
+                                         ],
+                                         metadata={key: self._track_metadata[key]
```

Comment:
overkill no? why not just `metadata=self._track_metadata`?

Reply:
yep